### PR TITLE
Add Item Panel changes

### PR DESCRIPTION
**Pull request by ezambra**
* Added a search text field when there are more than 20 work items in each work item type section.
* Using `DetailsList` component now, instead of `ScrollableList`. This is to fix the selection issue we had with using CTRL and remembering selection.

### CODE REVIEW

File: src/PortfolioPlanning/Components/Plan/AddItemPanel.scss [hunk 1]
```diff
@@ -64,12 +62,23 @@
     margin-top: 5px;
 }
 
+.workItemTypeSectionBody {
+    margin-left: 10px;
```
Comment (shutchings) [inline src/PortfolioPlanning/Components/Plan/AddItemPanel.scss]: In general we should try and use the defined styles by azure devops. They're on an 8px scale and you can use $spacing-8, $spacing-16 for consistency.

File: src/PortfolioPlanning/Components/Plan/AddItemPanel.tsx [hunk 2]
```diff
@@ -272,26 +276,163 @@ export class AddItemPanel extends React.Component<IAddItemPanelProps, IAddItemPa
...
+                Object.keys(content.items).forEach(workItemId => {
+                    const item = content.items[workItemId];
+                    allItemsCount++;
+
+                    if (item.hide === false) {
+                        listItems.push(item);
+                    }
+                });
+
+                const columns: IColumn[] = [
```
Comment (shutchings) [inline src/PortfolioPlanning/Components/Plan/AddItemPanel.tsx]: Is it worth putting the icon, id, and title all in separate columns?

Comment (ezambra) [reply src/PortfolioPlanning/Components/Plan/AddItemPanel.tsx]: if we wanted to sort by column, maybe... not needed for now

Comment (shutchings) [reply src/PortfolioPlanning/Components/Plan/AddItemPanel.tsx]: Makes sense. Good to know about.

File: src/PortfolioPlanning/Components/Plan/AddItemPanel.scss (hunk 1 above)
Comment (ezambra) [reply src/PortfolioPlanning/Components/Plan/AddItemPanel.scss]: Fixed... waiting for Pranoti's revert to merge this one.

**Pull request merged**